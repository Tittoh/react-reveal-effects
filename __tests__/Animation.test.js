/*
 * Animation Test Suite
 *
 * Copyright Â© Roman Nosov 2017
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */
import Animation from '../Animation';

import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe('Animation', () => {
  it('renders a initial view', () => {
    const content = shallow(
      <Animation steps={Animation.step("1", 1000).step("2", 2000)}>
        <div>Test test</div>
      </Animation>
    );
    expect(content.html()).toMatchSnapshot();
  });
});
