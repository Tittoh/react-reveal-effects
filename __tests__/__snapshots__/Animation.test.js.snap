// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Animation renders a initial view 1`] = `"<div>Test test</div>"`;
